src/components/quantity-input/quantity-input.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { MinusIcon, PlusIcon } from '../icons';
+import styles from './quantity-input.module.scss';
+import { useRef } from 'react';
+
+type QuantityInputProps = {
+    value: number | null;
```

Comment:
What's the reason for using null instead of zero?

Reply:
No reason, I see I can use zero